datalad/support/annexrepo.py

```diff
@@ -130,6 +130,13 @@ def __init__(self, path, url=None, runner=None,
         batch_size: int, optional
           if specified and >0, instructs annex to batch this many commands before
           annex adds acts on git repository (e.g. adds them them to index for addurl).
+
```

Comment:
Dunno, was following the present style.

In general, we follow many different conventions in different files. I am personally trying to imitate whatever I find.

I guess uniformization will be impossible (or will not last) until someone writes up or points to a set of style guidelines.

I am pretty much without emotions regarding the particular style we choose. but at the same time I don't see myself choosing ;-)